airbyte-integrations/bases/base-singer/base_singer/singer_helpers.py

```diff
@@ -134,7 +134,8 @@ def create_singer_catalog_with_selection(masked_airbyte_catalog, discovered_sing
         for singer_stream in discovered_singer_catalog.get("streams"):
             if singer_stream.get("stream") in stream_to_airbyte_schema:
                 new_metadatas = []
-
+                # support old style catalog.
+                singer_stream["schema"]["selected"] = True
```

Comment:
@jrhizor - i think this integration uses old style singer. I think this is the right place to fix this. i'm surprised i thought we had other integrations that did this.